src/script/common/helper.cpp

```diff
@@ -78,11 +79,16 @@ v3f LuaHelper::readParam(lua_State *L, int index)
 }
 
 template <>
-std::string LuaHelper::readParam(lua_State *L, int index)
+std::string_view LuaHelper::readParam(lua_State *L, int index)
```

Comment:
Perhaps add a comment concerning the lifetime / validity of this string view?

(refman: "Because Lua has garbage collection, there is no guarantee that the pointer returned by lua_tolstring will be valid after the corresponding Lua value is removed from the stack"; this is respected in the code that's currently using this, but I think it would be good to explicitly document that we're passing on this lifetime restriction)

Reply:
Question is where to put the comment. Since the template specialization is not defined in the header I think no IDE would show it.

Reply:
I would be fine with declaring the template specialization in the header for the purpose of adding such a comment.